### PR TITLE
Improve Docs

### PR DESCRIPTION
**Pull request by Ananda1113**
# Description

A short summary of what is included in your Pull Request.

Fixes #(issue_no)

Replace `issue_no` in the above line, with the issue related to this PR.

## Type of change

Choosing one or more options from the following as per the nature of your Pull request.
- NOTE: *These boxes can be checked using **`[X]`***

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation Update

# Checklist:
Please tick all the boxes that are fulfilled by your Pull Request.

- [ ] I have named my files and folder, according to this project's guidelines.
- [ ] My code follows the style guidelines of this project.
- [ ] My Pull Request has a descriptive title. (not a vague title like `Update index.md`)
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have created a helpful and easy to understand `README.md`, according to the given [`README_TEMPLATE.`](https://github.com/Python-World/Python_and_the_Web/blob/master/README_TEMPLATE.md)
- [ ] I have included a requirements.txt file (if external libraries are required.)
- [ ] My changes do not produce any warnings.
- [ ] I have added a working sample/screenshot of the script.

### CODE REVIEW

**Comment by AdityaJ7** [comment]
Hi @Ananda1113 Thanks for creating this PR, we currently don't plan to change the name of the Repo as it would create URL issues 
but we appreciate your efforts so instead of changing the Repo name you can add a line in under the Description heading 
like:- "An amazing project for python lovers."

Thanks!

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Python_and_the_Web
+# Python_and_the_Web_an_Amazing_Project
```
Comment (AdityaJ7) [inline README.md:1]: Change this back to original

Comment (AdityaJ7) [reply README.md:1]: Add this Line: **An amazing project for Python lovers**, after the description of project and before the Note.

**Comment by AdityaJ7** [comment]
Code with Harry😁 @Ananda1113 

He is doing a phenomenal job!!

**Comment by hugovk** [comment]
Please don't create lots of spam PRs for Hacktoberfest, it's against the rules and values, and not fair on maintainers.

https://hacktoberfest.digitalocean.com/details#spam

Maintainers: please label this `invalid` so it's not counted towards Hacktoberfest.

**Comment by AdityaJ7** [comment]
> Please don't create lots of spam PRs for Hacktoberfest, it's against the rules and values, and not fair on maintainers.
> 
> https://hacktoberfest.digitalocean.com/details#spam
> 
> Maintainers: please label this `invalid` so it's not counted towards Hacktoberfest.

Thanks @hugovk

**Pull request closed without merging**